english/lynnmp/lynns_additions.md

```diff
@@ -0,0 +1 @@
+When you feel triggered or agitated, just stop, close your eyes, and take a few big <a href="https://en.wikipedia.org/wiki/Meditation">breaths</a>.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.